makefile.shade

```diff
@@ -3,8 +3,21 @@ var VERSION='0.1'
 var FULL_VERSION='0.1'
 var AUTHORS='Microsoft Open Technologies, Inc.'
 
+default BASE_DIR='${Directory.GetCurrentDirectory()}'
+default TARGET_DIR='${Path.Combine(BASE_DIR, "artifacts")}'
+
 use-standard-lifecycle
 k-standard-goals
 
 #xml-docs-test .clean .build-compile description='Check generated XML documentation files for errors' target='test'
-  k-xml-docs-test
\ No newline at end of file
+  k-xml-docs-test
+
+#nuget-pack target='package'
+  @{
+  	var outputDir = Path.Combine(TARGET_DIR, "build");
```

Comment:
Question... Is the build output runnable? As far as I know, if you want a self contained app you need to publish it